### PR TITLE
Active Vulnerability Detection

### PR DESCRIPTION
**Pull request by dylankilkenny**

### CODE REVIEW

File: docs/modules/ROOT/pages/module/code.adoc [hunk 1]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+Smart Contract assets can be manually triggered for reports on the report page. GitHub assets can be triggered manually and automatically from new pull requests (PRs) and commits.
+
+image::code-assets.png[Code Assets]
+
+[[asset-settings]]
+=== Settings
+
+The Asset Settings page offers two main settings to manage how Defender handles asset security: Automatically Generate Report and Vulnerability Detection.
+
+==== Automatically Generate Report
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:150]: I suggest being more descriptive in the title and using "Automatically Trigger Report for Pull Requests"

File: docs/modules/ROOT/assets/images/code-assets.png
Comment (zanbel) [inline docs/modules/ROOT/assets/images/code-assets.png:1]: The "Auto Run" header doesn't align with the text below. I suggest using "Auto Trigger"

File: docs/modules/ROOT/pages/module/code.adoc [hunk 2]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
 
-The Settings page allows you to manage the permissions and access level of the Code Inspector.
+Automatically Generate Report, is specifically for GitHub repositories. When enabled, this setting ensures continuous monitoring by generating new reports automatically whenever a new pull request (PR) is opened or a new commit is pushed to a PR. 
 
-In the Repositories tab, you can view all the repositories where the app is currently installed, helping you track where the app is actively conducting code analysis. If you need to make changes to the repositories that the app has access to, a convenient link takes you directly to the GitHub settings page of the app, facilitating effortless repository management.
+==== Vulnerability Detection
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:154]: title "Active Vulnerability Detection and Notification"

File: docs/modules/ROOT/pages/module/code.adoc [hunk 3]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+
+Vulnerability Detection provides a robust security measure by automatically scanning your assets for new critical vulnerabilities as they are discovered. This setting is applicable to both GitHub repositories and smart contracts. To enable this feature, users must activate the setting and select a specific branch to be monitored for GitHub assets. Key features of Vulnerability Detection include:
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:156]: "Active Vulnerability Detection and Notification provides a robust security measure by automatically scanning your assets for new vulnerabilities. 
This setting is applicable to both GitHub repositories and smart contracts. To enable this feature for Github repositories assets, you must activate the setting and select a specific branch to be monitored."

File: docs/modules/ROOT/pages/module/code.adoc [hunk 4]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+
+* *Automated Scanning*: Leveraging Defender's advanced scanning capabilities, this feature detects vulnerabilities in smart contracts, whether they are disclosed or undisclosed. When our team becomes aware of a new vulnerability, we promptly update our scanning algorithms to detect it.
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:158]: Automated Scanning: Leveraging Defender's advanced smart contracts scanning capabilities, when our team becomes aware of a new vulnerability, whether disclosed or undisclosed, we promptly update our scanning algorithms to detect it. We then scan all assets configured to be monitored

File: docs/modules/ROOT/pages/module/code.adoc [hunk 5]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+* *Automated Alerts*: Upon detection of a vulnerability, automated email notifications are sent to account administrators, alerting them to the potential risk. The notifications include relevant information tailored to the nature of the vulnerability and may also inform administrators if their smart contracts are unaffected.
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:159]: Automated Notification: Once the automatic scanning is completed an automated system notification email will be sent to inform you whether the vulnerability was detected in your monitored assets. If detected, the notifications include relevant information tailored to the nature of the vulnerability.

File: docs/modules/ROOT/pages/module/code.adoc [hunk 6]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+* *Risk Mitigation*: To aid in addressing and mitigating detected vulnerabilities, notifications may include suggestions for risk mitigation, providing actionable steps for administrators to protect their smart contracts and associated assets.
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:160]: Risk Mitigation: When detected, to aid in addressing and mitigating detected vulnerabilities, notifications may include suggestions for risk mitigation, providing actionable steps for administrators to protect their smart contracts and associated assets.

File: docs/modules/ROOT/pages/module/code.adoc [hunk 7]
```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
...
+
+By enabling Vulnerability Detection, users benefit from continuous, automated scanning and timely notifications, empowering them to respond quickly to new threats and maintain the security of their smart contracts and GitHub repositories. This proactive approach ensures that users can stay ahead of potential vulnerabilities and safeguard their code effectively.
```
Comment (zanbel) [inline docs/modules/ROOT/pages/module/code.adoc:162]: By enabling Active Vulnerability Detection and Notification for your assets, you can benefit from continuous, automated scanning and timely notifications, empowering you to respond quickly to new threats and maintain the security of your smart contracts and GitHub repositories. This proactive approach ensures that you can stay ahead of potential vulnerabilities and safeguard your code effectively.

**Review by zanbel — CHANGES_REQUESTED** [review]
@dylankilkenny looks great! made some minor copy suggestions

**Review by shahnami — APPROVED** [review]
LGTM!

**Pull request merged**